vowpalwabbit/kernel_svm.cc

```diff
@@ -899,7 +899,7 @@ VW::LEARNER::base_learner* kernel_svm_setup(options_i& options, vw& all)
 
   params->all->weights.stride_shift(0);
 
-  learner<svm_params, example>& l = init_learner(params, learn, predict, 1);
+  learner<svm_params, example>& l = init_learner(params, learn, predict, 1, all.get_setup_name(kernel_svm_setup));
```

Comment:
not a fan of this solution but not sure what other options are there. it's a bit absurd that the information to get this name is right above it (options_group) but now everything is wired in a circular manner by again adding more state to the all object.

Reply:
The name returned by setup can be overridden ridden by parse_args.  So the options_group by itself is not enough.